tsconfig.json

```diff
@@ -17,7 +17,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react"
+    "jsx": "react",
+    "noImplicitAny": false
```

Comment:
Why do we need to disable the `noImplicitAny` rule? It's probably best to avoid having things with an implicit any type as that circumvents the type checking provided by TypeScript.

Reply:
@mknox1225 I looked into this a bit after we discussed. There's no type file for the icon library you used, but we can manually define one. The method described in the "Plan C" section of [this blog post](https://medium.com/@steveruiz/using-a-javascript-library-without-type-declarations-in-a-typescript-project-3643490015f3) should work. This won't actually give us any type info for the icon library, but it'll get rid of the error and allow us to not disable the "noImplicitAny" rule.